ui/app/clinical/consultation/mappers/specimenMapper.js

```diff
@@ -21,7 +21,7 @@ Bahmni.Clinical.SpecimenMapper = function () {
 
     this.mapSpecimenToObservation = function (specimen) {
         var observation = {};
-        observation.dateCollected = moment(specimen.dateCollected).format("YYYY-MM-DDTHH:mm:ss.SSSZ");
+        observation.dateCollected = moment(specimen.dateCollected).format("YYYY-MM-DD");
```

Comment:
any specific reason we are getting rid of the timestamp? time is also very relevant

Reply:
Functionally, this field is meant to capture the sample collection date. It is captured from a date field(calendar) from the UI so timestamp won't be relevant. Earlier this timestamp was causing timezone offset overhead while saving and retrieving it.